storage/innobase/row/row0import.cc

```diff
@@ -4154,31 +4176,41 @@ fil_tablespace_iterate(
 		iter.file_size = file_size;
 		iter.n_io_buffers = n_io_buffers;
 
+		size_t buf_size= (1 + iter.n_io_buffers)
+							<< srv_page_size_shift;
```

Comment:
This would fit in a single line if you wrote `* srv_page_size`. The rest of the code is already referring to `srv_page_size`, so it could save a load of another memory location. Most CPUs have instructions for fast integer multiplication.

Reply:
This was just copied from what was there before, but I updated it.